.travis.yml

```diff
@@ -186,7 +186,7 @@ before_install:
   - |
     if [ -n "${USE_DEBUG}" ]; then
         # see gh-10676; need to pin pytest version with debug
-        # Python 3.5
+        # Python 3.7
```

Comment:
I think this comment should indicate Python 3.6 after the revisions, but wasn't even worth flushing the CI for such a small thing IMO. Catch in the next cleanup.

Reply:
Thanks, good call Tyler - we should get in this habit more, merge first and flag minor things for follow-up. I have another CI PR to make today, so will take it along there.